utils/controleval.py

```diff
@@ -33,6 +35,11 @@
 SSG_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
 
 
+class setEncoder(json.JSONEncoder):
```

Comment:
I don't think this is necessary anymore? I was able to remove it and the stdout functionality worked fine.